runtime/aardvarks/info.yaml

```diff
@@ -0,0 +1,79 @@
+apiVersion: apigeeregistry/v1
+items:
+  - apiVersion: apigeeregistry/v1
+    kind: API
+    metadata:
+      name: apigee-apihub-demo-fauna-aardvarks-proxy
+      labels:
+        apihub-business-unit: apigee-apihub-demo
+        apihub-kind: proxy
+        source: generate-animal-apis
+      annotations:
+        apigee-proxy: apigee-apihub-demo/apis/fauna-aardvarks
+    data:
+      displayName: 'apigee-apihub-demo proxy: fauna-aardvarks'
```

Comment:
Is our need to concatenate org, type, and name into a display name field for lists? It worries me that we're doing this when using the real data with groupings and columns would be safer and more flexible. How do we expect that users will use this field?

Reply:
Stepping back a bit, names (ids) should be unique and stable, thus they are a bit long... the current display names are just matching this but in a more readable form. I think this is a workaround to limitations in our current display options in API Hub. A dedicated signal viewer (if one existed) might provide ways of viewing information that are more flexible.

Reply:
Yes, as we've defined it, "name" must be a primary key - so it will have to contain the source name (as a namespace) concatenated with anything that the source is using to make it's entries unique and stable. And name shouldn't be relied on by human viewers. I feel it remains a concern that the "display name" is arbitrary and subject to the whims of the signal source instead of the viewer in the app, but it will no doubt be useful to reader of the YAML anyway.

Reply:
Names of signal APIs are updated to include the generator name, e.g. `generate-animal-apis-apigee-apihub-demo-proxy-fauna-aardvarks`